Library/Formula/synergy.rb

```diff
@@ -0,0 +1,63 @@
+class Synergy < Formula
+  homepage "http://synergy-project.org"
+  url "https://github.com/synergy/synergy/archive/1.6.3.tar.gz"
+  sha256 "93b1965b8e0cfc55cc654aab7053a58b9e730d09e68bbc520be32353a01dce1a"
+
+  depends_on "cmake" => :build
+  depends_on "qt5" => :build
+
+  patch :DATA
```

Comment:
hum ok! 

is there a good tap I could push it to? 
or would it be worth having me push it to my own tap, until it's patched?